source/lib/cdk-textract-stack.ts

```diff
@@ -180,6 +182,12 @@ export class CdkTextractStack extends cdk.Stack {
         "Origin Access Identity for Textract web stack bucket cloudfront distribution",
     });
 
+    const edgeLambdaFunctionArn = new SsmParameterReader(this, 'LambdaFunctionArnReader', {
+      parameterName: "lambdaFunctionArnParameterName",
```

Comment:
I could see this parameter name not being "unique enough". Could we use something on the lines with DUSLambdaedge...